src/dsp_tools/models/langstring.py

```diff
@@ -95,7 +95,7 @@ def __getitem__(self, key: Optional[Union[Languages, str]] = None) -> str:
             else:
                 try:
                     return list(self._langstrs)[0]
-                except:
+                except:  # pylint: disable=bare-except
                     return None
```

Comment:
If returning `None` is possible, then the type hint is not correct. Also, this code is really bad and "low hinging fruit" to improve:

```python
            if self._simplestring is not None: # or even `if self._simplestring: return`
                return self._simplestring
            return list(self._langstrs)[0] if list(self._langstrs) else None # or probably even `... if self._langstrs else None`
```